firebase-database/src/main/java/com/google/firebase/database/tubesock/WebSocket.java

```diff
@@ -261,8 +261,8 @@ private synchronized void closeSocket() {
     if (socket != null) {
       try {
         socket.close();
-      } catch (IOException e) {
-        throw new RuntimeException(e);
+      } catch (Throwable e) {
```

Comment:
catching `Throwable` seems a little bit aggressive as it will also catch `java.lang.Error`s which, according to its javadoc, "a reasonable application should not try to catch". Would it be sufficient to only catch `Exception` for example?

Reply:
Fixed. I only mean to catch IOException and NPEs here.